setup.py

```diff
@@ -41,6 +41,7 @@
     packages=find_packages(exclude=['tests*', 'benchmarks*']),
     include_package_data=True,
     install_requires=[
+        'avro',
```

Comment:
if you install yelp-avro it actually creates a package with name `avro`. That is how we are able to import just avro. 
I am not completely happy with the way its implemented but couldn't think of any other neat way.
@jian-yelp thoughts?